pynio/block.py

```diff
@@ -31,7 +31,7 @@ def save(self):
         config['name'] = self._name
         config['type'] = self._type
         # load template and then reload config
-        self.template = self._instance.blocks_types[self._type].template
+        self.template = self._instance.blocks_types[self._type]
```

Comment:
This doesn't make sense. setting self.template to a Block? Isn't self.template supposed to be a Properties object?

It also concerns me that no unit tests caught this.

Not to mention, I think a minor refactoring task should happen at some point so that blocks_types are not Block objects but instead BlockType objects.